packages/twenty-server/src/engine/metadata-modules/database-event-trigger/services/database-event-trigger-v2.service.ts

```diff
@@ -47,6 +61,8 @@ export class DatabaseEventTriggerV2Service {
       fromCreateDatabaseEventTriggerInputToFlatDatabaseEventTrigger({
         createDatabaseEventTriggerInput: databaseEventTriggerInput,
         workspaceId,
+        workspaceCustomApplicationId:
```

Comment:
same here

Reply:
I though I've added jsdocs deprecated to this though